Add command options and documentation to Docker Compose

benjisho opened the pull request:
This PR introduces the command configuration for the bandersnatch service in Docker Compose setup. It includes the default 'mirror' command and adds commented options for 'verify' and 'once' to provide alternatives for running bandersnatch.

The 'command' section is now clearly documented, making it easier for developers and sysadmins to understand and modify the behavior of the bandersnatch service.

Changes:
- Added 'command' directive to bandersnatch service with the default 'mirror' option.
- Included comments explaining how to use the 'verify' and 'once' commands.
- Explained the purpose of additional flags for the 'mirror' command.
- Provided documentation for the build context of the bandersnatch_nginx image.

cooperlees reviewed (changes requested):
Sounds good - Can you add some examples of how to use this into the Docker compose readme please?

https://github.com/pypa/bandersnatch/blob/main/src/bandersnatch_docker_compose/README.md

Might help someone like me who's never ran `docker-compose` before :)

benjisho commented:
Changes:

- Added command options for bandersnatch service in docker-compose.yml
- Added "Removing the Repository" section in Docker Compose README.md
- Added link in the main README.md to the Docker Compose section

Users can now choose from several modes by uncommenting the corresponding command in the docker-compose.yml file.

Please review and let me know if any changes are required.
Users can now choose from several modes by uncommenting the corresponding command in the docker-compose.yml file.

cooperlees commented:
Can you just run `pre-commit` and format the .md files please.

benjisho commented:
> Can you just run `pre-commit` and format the .md files please.

`pre-commit` done

cooperlees reviewed (approved):
Awesome. Many thanks for this and battling through our CI :)

It's always a learning curve with each project one contributes too.
